apps/frontend/src/pages/CreateCharity.tsx

```diff
@@ -0,0 +1,671 @@
+import { ChangeEvent, FormEventHandler, useState } from "react";
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Checkbox,
+  Container,
+  FormControl,
+  FormControlLabel,
+  Grid,
+  Paper,
+  TextField,
+  Typography,
+} from "@mui/material";
+import SearchBar from "../components/searchBar";
+import { colors } from "../styles/colors";
+import _axios_instance from "../_axios_instance";
+
```

Comment:
We keep defining these Organization/Charity objects. We could either implement it in one place like types.ts or my PR that I just merged has them defined in Charities.tsx which you should be able to use from there. It has the `export` keyword so you should be able to use it in your files. 

I do use the Charities object in several files and use the exported type from Charities.tsx

Reply:
Those charity objects are now in the types file!